### PR TITLE
API for Question Answer and Subject

### PR DESCRIPTION
**Pull request by yashpuro**

### CODE REVIEW

File: exam/views.py [hunk 1]
```diff
@@ -15,10 +15,20 @@ def admin(request):
...
-def SubjectViewSet(ModelViewSet, pk):
-    queryset = Subject.objects.filter(semester=pk).values()
+def SubjectViewSet(ModelViewSet, pk , stream):
+    queryset = Subject.objects.filter(semester=pk , stream = stream).values()
     serializer_class =  SubjectSerializer(queryset, many=True)
-    #permisssion_classes = [permissions.IsAuthenticated] 
-    return JsonResponse(serializer_class.data, safe=False)    
+    return JsonResponse(serializer_class.data, safe=False)  
+
+def QuestionViewSet(ModelViewSet, pk ):
```
Comment (suman-kr) [inline exam/views.py]: Use class based views

**Review by su-gu — APPROVED** [review]

File: exam/views.py [hunk 2]
```diff
@@ -15,10 +15,20 @@ def admin(request):
...
+    queryset = Question.objects.filter(pk=pk)
+    serializer_class =  QuestionSerializer(queryset, many=True)
+    return JsonResponse(serializer_class.data, safe=False)
```
Comment (suman-kr) [inline exam/views.py]: Use Response from DRF https://www.django-rest-framework.org/tutorial/2-requests-and-responses/

Comment (suman-kr) [reply exam/views.py]: Read: https://stackoverflow.com/questions/47192986/difference-between-response-and-httpresponse-django

File: exam/views.py [hunk 3]
```diff
@@ -15,10 +15,20 @@ def admin(request):
...
     return HttpResponse(template.render(context, request))
+   
 
 
-def SubjectViewSet(ModelViewSet, pk):
-    queryset = Subject.objects.filter(semester=pk).values()
+def SubjectViewSet(ModelViewSet, pk , stream):
+    queryset = Subject.objects.filter(semester=pk , stream = stream).values()
     serializer_class =  SubjectSerializer(queryset, many=True)
-    #permisssion_classes = [permissions.IsAuthenticated]
```
Comment (suman-kr) [inline exam/views.py:23]: Remove comments

**Comment by suman-kr** [comment]
**Write description for your PR**

**Comment by suman-kr** [comment]
https://www.django-rest-framework.org/api-guide/responses/

**Pull request merged**